app/controllers/search.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
should this file be called `get-all-posts.php` or do you envisage other functions being in here